src/compiler/generate-common-tests.xsl

```diff
@@ -300,16 +302,18 @@
    -->
    <xsl:template match="x:expect" mode="x:generate-calls">
       <xsl:param name="pending" select="()" tunnel="yes" as="node()?"/>
-      <xsl:param name="vars"    select="()" tunnel="yes" as="element(var)*"/>
+      <xsl:param name="vars"    select="()" tunnel="yes" as="element(x:var)*"/>
       <xsl:call-template name="x:output-call">
          <xsl:with-param name="local-name" select="generate-id()"/>
          <xsl:with-param name="last" select="empty(following-sibling::x:expect)"/>
          <xsl:with-param name="params" as="element(param)*">
             <xsl:if test="empty($pending|ancestor::x:scenario/@pending) or exists(ancestor::*/@focus)">
-               <param name="{x:xspec-name('result')}" select="${x:xspec-name('result')}" />
+               <param name="{x:xspec-name(.,'result')}" select="${x:xspec-name(.,'result')}" />
             </xsl:if>
             <xsl:for-each select="x:distinct-variable-names($vars)">
-               <param name="{ . }" select="${ . }"/>
+               <param name="{ @name }" select="${ @name }">
+                  <xsl:sequence select="x:copy-namespaces(.)"/>
+              </param>
```

Comment:
```suggestion
               </param>
```

Reply:
Fixed in 21ed0e1f844324b13670d7ad7828bb9894d2a26f